### PR TITLE
feat: paraswap withdrawal aggregator

### PR DESCRIPTION
**Pull request by manneredboor**
### Description

ParaSwap withdrawal aggregator returned

### Demo

image

### Checklist:

- [x] Checked the changes locally.

### CODE REVIEW

**Review by DiRaiks — APPROVED** [review]

**Review by DiRaiks — CHANGES_REQUESTED** [review]
I did not find the ParaSwap URL added to the CSP trusted URLs. This must be done to avoid false positives.

**Comment by manneredboor** [comment]
> I did not find the ParaSwap URL added to the CSP trusted URLs. This must be done to avoid false positives.

It was not deleted before, still here: https://github.com/lidofinance/ethereum-staking-widget/blob/feature/si-1177-return-paraswap-to-aggregators-page/utilsApi/withCSP.ts#L55 @DiRaiks

**Review by DiRaiks — APPROVED** [review]

**Review by DiRaiks — CHANGES_REQUESTED** [review]
Waiting for confirmation of ParaSwap link

File: features/withdrawals/request/form/options/dex-options.tsx [hunk 1]
```diff
@@ -42,6 +43,18 @@ const dexInfo: {
...
       )}#/ETH/${token}/ETH`,
   },
+  paraswap: {
+    title: 'ParaSwap',
+    icon: <ParaSwapIcon />,
+    onClickGoTo: () => {
+      trackMatomoEvent(MATOMO_CLICK_EVENTS_TYPES.withdrawalGoToParaswap);
+    },
+    link: (amount, token) =>
+      `https://app.paraswap.io/#/${getTokenAddress(
```
Comment (DiRaiks) [inline features/withdrawals/request/form/options/dex-options.tsx:53]: Need change link to 
```ts
 `https://app.paraswap.io/referrer=Lido&takeSurplus=true#/${getTokenAddress(
        CHAINS.Mainnet,
        token,
      )}-ETH/${formatEther(amount)}/SELL?network=ethereum`,
 ```

Comment (manneredboor) [reply features/withdrawals/request/form/options/dex-options.tsx]: Done

**Review by DiRaiks — DISMISSED** [review]

**Review by jake4take — APPROVED** [review]

**Review by Jeday — APPROVED** [review]

**Pull request merged**